core/src/main/scala/cats/Eval.scala

```diff
@@ -310,6 +310,30 @@ private[cats] trait EvalInstances extends EvalInstances0 {
       }
     }
 
+  implicit val catsReducibleForEval: Reducible[Eval] =
```

Comment:
should this be combined with the monad/comonad above?

I don't *think* we can ever implement `Traverse[Eval]` (I'd love to be wrong about that), so it didn't have the same ambiguous implicit concern that `Id` has (since `Traverse[Id]` also extends `Foldable[Id]`, we don't want `Reducible[Id]` we have to either use priority, or unify them in the same instance).